lib/rules/global-require.js

```diff
@@ -52,9 +52,25 @@ function isShadowed(scope, node) {
 /** @type {import('../shared/types').Rule} */
 module.exports = {
     meta: {
-        deprecated: true,
-
-        replacedBy: [],
+        deprecated: {
+            message: "Node rule were moved out of ESLint core.",
```

Comment:
Maybe "Node.js" sounds better than just "Node" here. More importantly, the sentence should read "Node.js rule**s** were ..." or "Node.js rule **was** ...". We should correct this message in multiple places.